Mailbox/Mailbox.cs

```diff
@@ -1,4 +1,52 @@
-﻿namespace Mailbox
+﻿using System;
+using System.Collections;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Mailbox
 {
+    // I AM A MAIL BOX
+    public class Mailbox : IEquatable<Mailbox>
+    {
+        //mailbox.Location == (x, y)
+        public (int X, int Y) Location { get; set; }
+
+        public Sizes BoxSize{ get; set; } 
+
+        public Person Owner { get; set; }
+
+        public Mailbox(Sizes size, ValueTuple<int, int> location, Person Owner)
+        {
+            if (size.Equals(Sizes.Premium)) throw new ArgumentException(" \"Premium\" is not a size itself", (nameof(size)));
+
+            BoxSize = size;
+            Location = location;
+            this.Owner = Owner;
+        }
+
+        override
+    public string ToString()
+        {
+            string mailboxString = $"Owner: {Owner.ToString()}, Location: x = {Location.X}, y = {Location.Y}";
```

Comment:
Typically inside of an interpolated string you do not need to explicitly call ToString(), it will convert to string for you.
```suggestion
            string mailboxString = $"Owner: {Owner}, Location: x = {Location.X}, y = {Location.Y}";
```